package.json

```diff
@@ -4,24 +4,27 @@
   "main": "index.ts",
   "type": "module",
   "scripts": {
-    "build": "rimraf dist && tsc && rollup -c rollup.config.js && grep -v \"export { main }\" dist/index.js >> dist/action.js && rm dist/index.js",
```

Comment:
Does one of the rollup config changes remove the need for removing the export?

Reply:
yes - actually we have been using `commonjs` modules for a little while, so `export { main }` hasn't been in the built code since pre-release, this is just a leftover